unit3-server-mvc-with-node-express-and-sql/README.md

```diff
@@ -2,50 +2,58 @@
 
 ### Week 1
 
+- Serverside MVC with Express: 3 days
+  - Why MVC for server side development: Separation of concerns
+  - Routing & (simple) regular expressions
+  - Controllers
+  - Views using a templating engine and ViewModels (Mustache? Jade?)
```

Comment:
Yeah just meant 'one of the many.' Express has a built in way to set a 'view engine.' The Express application generator uses Pug by default, but I've never actually used Pug personally.... I've mostly used Jade, and a little bit of mustache. Both work well with expres